Runtime/Behaviours/CinemachineDollyCart.cs

```diff
@@ -59,6 +71,8 @@ void FixedUpdate()
 
         void Update()
         {
+            m_PathCache.InvalidateDistanceCache(); 
```

Comment:
I don't think you want to invalidate every frame!  That kills the whole value of caching.

Reply:
Yes you are right. I am waiting for callback() from spline team. It is in PR already. After I get it I will do something like 
`if (spline.change) InvalidateDistanceCache ()` here